src/main/java/com/danielagapov/spawn/Services/FriendTag/FriendTagService.java

```diff
@@ -49,4 +49,14 @@ public FriendTag saveFriendTag(FriendTag friendTag) {
             throw new BaseSaveException("Failed to save friendTag: " + e.getMessage());
         }
     }
+
+    public FriendTag replaceFriendTag(FriendTag newFriendTag, Long id) {
+        return repository.findById(id).map(friendTag -> {
```

Comment:
Btw, this is a suggestion, but another way of writing a method like this that either inserts/updates a thing based on whether it already exists, is to name it `upsert`. I won't block this current PR, but in the future, I'd like if we abstracted this logic out to a method called `upsertFriendTag` that has this finding if/else

Reply:
Then, we can re-use it more, and de-clutter these individual methods. But, it's just a styling preference, so it's not a huge deal.